mbed_greentea/mbed_greentea_cli.py

```diff
@@ -299,6 +347,10 @@ def main():
                     dest='run_app',
                     help='Flash, reset and dump serial from selected binary application')
 
+    parser.add_option('', '--testlink-test-plan',
```

Comment:
This can be called just ```--test-plan```. Since this is an input its format will be specified by Greentea.